README.md

```diff
@@ -140,24 +140,36 @@ kubectl apply -f https://raw.githubusercontent.com/fluent/fluentbit-operator/mas
 The Fluent Bit section of the Fluent Operator supports different CRI `docker`, `containerd`,  and `CRI-O`. 
 `containerd` and `CRI-O` use the `CRI Log` format which is different with `docker`, they requires additional parser to parse JSON application logs. You should set different `containerRuntime` depending on your container runtime.
 
-If your container runtime is `docker`
+If your container runtime is `docker`:
 
 ```shell
-helm install fluent-operator  --create-namespace -n kubesphere-logging-system charts/fluent-operator/ --set containerRuntime=docker
+helm install fluent-operator  --create-namespace -n kubesphere-logging-system charts/fluent-operator/ 
 ```
 
-If your container runtime is `containerd`
+If your container runtime is `containerd`:
 
 ```shell
 helm install fluent-operator --create-namespace -n kubesphere-logging-system charts/fluent-operator/  --set containerRuntime=containerd
 ```
 
-If your container runtime is `cri-o`
+If your container runtime is `cri-o`:
 
 ```shell
 helm install fluent-operator --create-namespace -n kubesphere-logging-system charts/fluent-operator/  --set containerRuntime=crio
 ```
 
+Install through the online chart link by releases:
+
+```shell
+helm install fluent-operator --create-namespace -n kubesphere-logging-system https://github.com/fluent/fluent-operator/releases/download/v1.0.0-rc.0/fluent-operator.tgz  --set containerRuntime=docker
+```
+
+Install through the online chart link by master branch:
+
+```shell
+helm install fluent-operator --create-namespace -n kubesphere-logging-system https://github.com/fluent/fluent-operator/blob/master/charts/fluent-operator.tgz  --set containerRuntime=docker
```

Comment:
I don't think this works either:
```
$ helm install fluent-operator --create-namespace -n kubesphere-logging-system https://github.com/fluent/fluent-operator/blob/master/charts/fluent-operator.tgz
Error: INSTALLATION FAILED: file '/home/pat/.cache/helm/repository/fluent-operator.tgz' does not appear to be a gzipped archive; got 'text/html; charset=utf-8'
```
You can get a tarball of a branch via the /tarball endpoint

Reply:
I updated the latest tgz file.

Reply:
Not sure I follow, specifically the URL specified here is not a valid chart for me: https://github.com/fluent/fluent-operator/blob/master/charts/fluent-operator.tgz
```
$ kind create cluster
Creating cluster "kind" ...
 ✓ Ensuring node image (kindest/node:v1.21.1) 🖼
 ✓ Preparing nodes 📦  
 ✓ Writing configuration 📜 
 ✓ Starting control-plane 🕹️ 
 ✓ Installing CNI 🔌 
 ✓ Installing StorageClass 💾 
Set kubectl context to "kind-kind"
You can now use your cluster with:

kubectl cluster-info --context kind-kind

Not sure what to do next? 😅  Check out https://kind.sigs.k8s.io/docs/user/quick-start/
$ helm install fluent-operator --create-namespace -n kubesphere-logging-system https://github.com/fluent/fluent-operator/blob/master/charts/fluent-operator.tgz
Error: INSTALLATION FAILED: file '/home/pat/.cache/helm/repository/fluent-operator.tgz' does not appear to be a gzipped archive; got 'text/html; charset=utf-8'
```
The URL is not one you can use that way:
```
$ wget https://github.com/fluent/fluent-operator/blob/master/charts/fluent-operator.tgz
$ tar -xzvf fluent-operator.tgz 
gzip: stdin: not in gzip format
tar: Child returned status 1
tar: Error is not recoverable: exiting now
$ file fluent-operator.tgz 
fluent-operator.tgz: HTML document, UTF-8 Unicode text, with very long lines
```

Reply:
I think you want the RAW content: https://github.com/fluent/fluent-operator/blob/master/charts/fluent-operator.tgz?raw=true

Reply:
This would work after this pr merged. 

I simply tried it:

```
root@allinone:~/go/src/github.com/fluentbit-operator#  helm install fluent-operator --create-namespace -n kubesphere-logging-system ./charts/fluent-operator.tgz
NAME: fluent-operator
LAST DEPLOYED: Tue Mar 15 19:41:57 2022
NAMESPACE: kubesphere-logging-system
STATUS: deployed
REVISION: 1
TEST SUITE: None
NOTES:
Thank you for installing  fluent-operator
Your release is named    fluent-operator

To learn more about the release ,try:
   $ helm status  fluent-operator  -n  kubesphere-logging-system
   $ helm get  fluent-operator  -n kubesphere-logging-system
```

Reply:
Oh right, I assumed you were grabbing the branch directly rather than a binary on it. Does this mean every push to master now has to update the binary too or is it automated?

Reply:
> Oh right, I assumed you were grabbing the branch directly rather than a binary on it. Does this mean every push to master now has to update the binary too or is it automated?

It can be automated. We add a cmd "make update-helm-package". I think this part can be automated if needed through a CI action.

Reply:
The `master` one is not really `master` then, it's just whatever got merged from the last PR. This seems wrong to me and another reason just to remove it. There are already URLs to get a tarball from any Github branch but honestly people should either be using a "release" version or cloning the repo and using locally: I don't think we should expose this `master` tarball.

Reply:
Let's remove it.